srv/SetCommandLifetime.srv

```diff
@@ -0,0 +1,2 @@
+float64 duration
```

Comment:
question -- what about `duration_ms` or `duration_s` instead?  Same with frequency below -- perhaps append `hz`?  Alternatively, we could document units here with a comment.

Reply:
I switched duration to the relevant ROS type (duration), and added hz to the frequency service